### DISCUSSION TITLE
No /plugins/jspsych-text.js found

### DISCUSSION THREAD

**Discussion by aksylumoed** [jspsych/jsPsych#485] [Q&A]
The .js for text (as shown in the documentation tutorial) is not found in the repo.

**Comment by jodeleeuw** [comment]
You can find it in the [tag for version 5.0.3](https://github.com/jspsych/jsPsych/tree/v5.0.3). 

The jspsych-text plugin no longer exists in the development version because it was mostly redundant with the single-stim plugin. Both of those plugins have been removed and replaced with the html-keyboard-response and image-keyboard-response plugins.
